spec/linter-eslint-spec.js

```diff
@@ -84,23 +84,28 @@ async function getNotification(expectedMessage) {
 }
 
 async function makeFixes(textEditor) {
-  return new Promise(async (resolve) => {
-    // Subscribe to the file reload event
-    const editorReloadSub = textEditor.getBuffer().onDidReload(async () => {
-      editorReloadSub.dispose()
-      // File has been reloaded in Atom, notification checking will happen
-      // async either way, but should already be finished at this point
+  const editorReloadPromise = new Promise((resolve) => {
+    // Subscribe to file reload events
+    const editorReloadSubscription = textEditor.getBuffer().onDidReload(() => {
+      editorReloadSubscription.dispose()
       resolve()
     })
+  })
 
-    // Now that all the required subscriptions are active, send off a fix request
-    atom.commands.dispatch(atom.views.getView(textEditor), 'linter-eslint:fix-file')
-    const expectedMessage = 'Linter-ESLint: Fix complete.'
-    const notification = await getNotification(expectedMessage)
+  const expectedMessage = 'Linter-ESLint: Fix complete.'
+  // Subscribe to notification events
+  const notificationPromise = getNotification(expectedMessage)
 
-    expect(notification.getMessage()).toBe(expectedMessage)
-    expect(notification.getType()).toBe('success')
-  })
+  // Subscriptions now active for Editor Reload and Message Notification
+  // Send off a fix request.
+  atom.commands.dispatch(atom.views.getView(textEditor), 'linter-eslint:fix-file')
+
+  const notification = await notificationPromise
```

Comment:
Wow, gotta love when solutions like this are staring you right in the face. Would you mind either submitting a PR fixing the other uses of `getNotification`, or just fixing them here?

Reply:
Yeah. I'm trying to start looking harder at the actual application code. But if you think this fix is likely to be applicable in other places in the specs, I'll definitely try to hunt them down.

Reply:
This one seems to be good to go as-is and affects multiple tests. So let's commit it, and I'll keep digging for more of the same.